### PR TITLE
Add current and expect jsons with transformer for edx.drag_and_drop_v2.feedback.opened

### PR DESCRIPTION
**Pull request by HamzaFarooqArbisoft**
**Trello Link:** _https://trello.com/c/7um9UQN1/74-drag-and-drop-event-edxdraganddropv2feedbackopened_

**Description:** _The server emits this event when a pop up feedback message opens in a drag and drop problem._

**Checks before merge:**

- [ ] Reviewed
- [ ] Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/tests/expected/edx.drag_and_drop_v2.feedback.opened.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "OpenedPopout",
```
Comment (asamolion) [inline openedx/features/caliper_tracking/tests/expected/edx.drag_and_drop_v2.feedback.opened.json]: I believe a `Viewed` event is more appropriate here. `OpenedPopout` is specific to video popouts.

File: openedx/features/caliper_tracking/caliper_config.py [hunk 2]
```diff
@@ -45,7 +45,6 @@
     'edx.forum.thread.created': ctf.edx_forum_thread_created,
     'openassessmentblock.get_peer_submission':
         ctf.openassessmentblock_get_peer_submission,
-    'openassessmentblock.peer_assess': ctf.openassessmentblock_peer_assess,
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/caliper_config.py]: Why are you deleting this value ?

File: openedx/features/caliper_tracking/transformers/__init__.py [hunk 3]
```diff
@@ -73,3 +73,6 @@
     openassessmentblock_create_submission,
     openassessment_student_training_assess_example,
 )
+from .drag_and_drop_transformers import (
+    edx_drag_and_drop_v2_feedback_opened,
+)
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/transformers/__init__.py]: Please add a newline here.

File: openedx/features/caliper_tracking/caliper_config.py (hunk 2 above)
Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/caliper_config.py]: It was a duplicate entry.

File: openedx/features/caliper_tracking/tests/expected/edx.drag_and_drop_v2.feedback.opened.json (hunk 1 above)
Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/tests/expected/edx.drag_and_drop_v2.feedback.opened.json]: done

File: openedx/features/caliper_tracking/transformers/__init__.py (hunk 3 above)
Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/transformers/__init__.py]: done

**Pull request merged**